docs/server/source/quickstart.md

```diff
@@ -28,6 +42,15 @@ E. Install the `bigchaindb` Python package from PyPI:
 $ sudo pip3 install bigchaindb
 ```
 
+In case you are having problems with installation or package/module versioning, please upgrade the relevant packages on your host by running one the following commands:
+```text
+$ sudo pip install [packageName]==[packageVersion]
+
+OR
+
+sudo pip3 install [packageName] --upgrade
```

Comment:
Add a $ in front for consistency on this page. (We're inconsistent on this in general, but should be consistent within the page.)